app/assets/javascripts/client/inventories/manageInventoryPermissions.js

```diff
@@ -0,0 +1,18 @@
+(function () {
+  'use strict';
+  angular.module('PDRClient')
+      .directive('manageInventoryPermissions', manageInventoryPermissions);
+
+  function manageInventoryPermissions() {
+    return {
+      restrict: 'E',
+      scope: {
+        inventoryId: '=inventoryId'
```

Comment:
Since the name of the variable here is the same, it should read `inventoryId: '='` instead.